docs/sender_profile/validation.md

```diff
@@ -0,0 +1,31 @@
+### `username`
+- Must be at least 3 characters long.
+- Cannot be more than 100 characters long.
+
+### `password`
+- Must be at least 6 characters long.
+- Cannot be more than 25 characters long.
+
+### `smtp_host`
+- Must be at least 5 characters long.
+- Cannot be more than 100 characters long.
+- Must be a valid host name (e.g., `example.com`).
+- It can either be a domain name or an ip address
+
+### `smtp_port`
+- Must be a valid port number (only digits).
```

Comment:
between 0 and 65535

Reply:
I implemented it for 65535, documented it wrong 🤦🏻
Will make the necessary changes.